nexus/lib/qmcpack_converters.py

```diff
@@ -185,6 +185,91 @@ def generate_pw2qmcpack_input(prefix='pwscf',outdir='pwscf_output',write_psir=Fa
 #end def generate_pw2qmcpack_input
 
 
+def import_nexus_module(module_name):
+    import importlib
+    return importlib.import_module(module_name)
+#end def import_nexus_module
+
+
+def read_eshdf_eig_data(filename, Ef_list):
+    import numpy as np
+    from numpy import array,pi
+    from numpy.linalg import inv
+    from unit_converter import convert
+    try:
+        hdfreader = import_nexus_module('hdfreader')
+        read_hdf = hdfreader.read_hdf
+        del hdfreader
+    except:
+        from hdfreader import read_hdf
+    #end try
+
+    def h5int(i):
+        return array(i,dtype=int)[0]
+    #end def h5int
+
+    h        = read_hdf(filename,view=True)
+    axes     = array(h.supercell.primitive_vectors)
+    kaxes    = 2*pi*inv(axes).T
+    nk       = h5int(h.electrons.number_of_kpoints)
+    ns       = h5int(h.electrons.number_of_spins)
+    if len(Ef_list) != ns:
+      msg = 'Ef "%s" must have same length as nspin=%d' % (str(Ef_list), ns)
```

Comment:
Why not just `error()` here instead of returning string?

Reply:
This stand-alone routine `gcta_occupation` is not part of any class, so does not have access to the `error` function.
Does it make sense to make it part of `Pw2qmcpackAnalyzer`?

Reply:
*`read_eshdf_eig_data` is stand-alone

Reply:
You can just import `error` from `developer`.  The class level error function wraps this one.

Reply:
gotcha fix in next push